mlrun/db/httpdb.py

```diff
@@ -55,10 +55,23 @@ def bool2str(val):
     return "yes" if val else "no"
 
 
+HTTP_RETRY_AMOUNT = 3
+HTTP_RETRY_BACKOFF = 1
```

Comment:
```suggestion
HTTP_RETRY_BACKOFF_FACTOR = 1
```
Currently it looks like it's a 0/1 selection whether to backoff or not. Better to clarify it is the factor.